webapp/src/css/inbox.less

```diff
@@ -667,13 +667,22 @@ body {
   border: 1px solid @separator-color;
   border-top: 0;
   border-bottom: 0;
+  .padding {
+    height: 120px;
+  }
+}
 
+.inbox-items, .targets {
```

Comment:
IMHO, if you decide to go with a dedicated component for the error alert, it would make sense to break this out into its own less file. Seems kinda weird to find `.targets` styling in the `inbox.less`...